docs/product/dev-toolbar/setup.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: "Set Up"
+sidebar_order: 10
+description: "Get started with Sentry's Dev Toolbar, bringing critical Sentry insights and tools into your web app to help your team troubleshoot more effectively."
+---
+
+<Alert level="info">
+  The Dev Toolbar is currently in **beta**. Beta features are still in progress and may have bugs. Please reach out on
+  [GitHub](https://github.com/getsentry/sentry-toolbar/issues) if you have any feedback or concerns.
+</Alert>
+
+## Set Up Your Web App
+
+<Alert title="Note" level="warning">
```

Comment:
IMO we could drop the title attr. 

Looks like this currently:

<img width="789" alt="SCR-20241213-lolg" src="https://github.com/user-attachments/assets/de7920d0-0730-4c25-a376-6e766202b21b" />